packages/applications-service-api/src/services/application.service.js

```diff
@@ -4,14 +4,40 @@ const getApplication = async (id) => {
 	return await db.Project.findOne({ where: { CaseReference: id } });
 };
 
-const getAllApplications = async (query) => {
+const createQueryFilters = (query) => {
+	// Pagination
 	const pageNo = parseInt(query?.page) || 1;
 	const size = Math.min(parseInt(query?.size) || 25, 100);
 
+	// Sorting
+	const allowedSortFieldsWithDirection = [
+		'ProjectName',
+		'PromoterName',
+		'DateOfDCOSubmission',
+		'ConfirmedDateOfDecision',
+		'Stage'
+	].flatMap((field) => [`+${field}`, `-${field}`]);
+
+	const sort = allowedSortFieldsWithDirection.includes(query?.sort) ? query?.sort : '+ProjectName';
```

Comment:
maybe assign `+ProjectName` to a variable (called `defaultSortField` or something), to make it obvious what that is for?